src/pages/createCohort/components/cohortList.jsx

```diff
@@ -0,0 +1,30 @@
+import {  Card } from "@mui/material";
+import { useEffect } from "react";
+import { useState } from "react";
+import client from "../../../utils/client";
+import "./style.css"
+
+const CohortList=()=>{
+    const [cohorts,setCohorts]=useState([])
+
+    useEffect(()=>{
+        client.get('/cohort').then((res)=>{setCohorts(res.data.data.cohorts)}).catch(console.log)
```

Comment:
I'm not sure if this is github displaying the code a little weirdly, but could you try running your formatter on this file just in case